modules/source/mail/config/mail.xml

```diff
@@ -34,7 +38,7 @@
 			store-uri="${protocol}://${username:}:${password:}@${host}:${port}/${folder}"
```

Comment:
For some reason I couldn't get the `imap-idle-channel-adapter working when I set --usePolling=false (which is by default false). The Polling mail source works.